src/App.jsx

```diff
@@ -1,20 +1,26 @@
 /* eslint-disable import/no-named-as-default */
 import React from 'react';
 import './App.scss';
+import { Provider } from 'react-redux';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import HeaderSection from './commons/components/HeaderSection';
 import FooterSection from './commons/components/FooterSection';
 import Landing from './Landing/containers/Landing';
+import AllArticles from './Articles/components/AllArticles';
+import store from './Mainstore/Store';
 
 export default function App() {
   return (
-    <Router>
+    <Provider store={store}>
+      <Router>
       <HeaderSection />
       <Switch>
         <Route path="/" exact component={Landing} />
         <Route path="/about" component={Landing} />
+        <Route path="/article" component={AllArticles} />        
```

Comment:
@neelxie set the path to `/articles` instead of `/article`

Reply:
@alexxsanya fixed, thank you.